sdk/cosmos/microsoft-azure-cosmos-benchmark/src/main/java/com/azure/data/cosmos/benchmark/ReadMyWriteWorkflow.java

```diff
@@ -90,12 +95,15 @@ protected void performWorkload(BaseSubscriber<Document> baseSubscriber, long i)
                     throw new IllegalStateException();
             }
         } else {
+
             // will either do
             // a write
-            // a point read for a in memory cached document
+            // a point read for a in memory cached document4
```

Comment:
should be `document` instead of `document4`

Reply:
thanks for finding this. it is now corrected.